### PR TITLE
CDAP-13570 Add Apache public repository, and remove Twill dependency workaround

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA: https://issues.cask.co/browse/CDAP-13570
Build: https://builds.cask.co/browse/CDAP-RUT1584-6

### CODE REVIEW

**Review by sreevatsanraman — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -57,6 +57,10 @@
   </scm>
 
   <repositories>
+    <repository>
+      <id>apache</id>
+      <url>https://repository.apache.org/content/repositories/public/</url>
```
Comment (anwar6953) [inline pom.xml:62]: Which repository was it getting the dependencies from previously? (where is that defined)

Comment (CuriousVini) [inline pom.xml:62]: Why this change was not needed in previous releases?

Comment (chtyim) [reply pom.xml:62]: The maven central http://repo2.maven.org/ (default, no need to specify).

Comment (poornachandra) [reply pom.xml:62]: @CuriousVini Details about this change is in the JIRA - https://issues.cask.co/browse/CDAP-13570?focusedCommentId=32916&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-32916

**Comment by poornachandra** [comment]
Build is green, merging...
